daliuge-translator/test/dropmake/test_pg_gen.py

```diff
@@ -162,3 +165,53 @@ def test_pg_eagle(self):
             fp = get_lg_fname(lg)
             lg = LG(fp)
             lg.unroll_to_tpl()
+
+    def test_cwl_translate(self):
+        import git
+        import os
+        import shutil
+        import uuid
+        import zipfile
+        import subprocess
+
+        output_list = []
+
+        cwl_output = "/tmp/cwloutput/"
```

Comment:
Not horribly important, but instead of using `"/tmp"` and hardcoded subdirectories there, it would be nicer to use `tempfile` and its methods to get temporary file/directory names, which then you'd remove at the end of the test (or otherwise the OS will automatically remove later on too).

Reply:
I'll make this change too.

Reply:
I've updated the code to use tempfile.mkdtemp() to create temporary directories for the CWL output and the clone of EAGLE_test_repo for input. Also, I shifted the existing use of shutil.rmtree() to delete the temporary directories to after the tests.